src/main/scala/is/hail/expr/Parser.scala

```diff
@@ -533,7 +533,7 @@ object Parser extends JavaTokenParsers {
       "Float" ^^ { _ => TFloat } |
       "Double" ^^ { _ => TDouble } |
       "String" ^^ { _ => TString } |
-      "Sample" ^^ { _ => TSample } |
+      "Sample" ^^ { _ => TString } | // FIXME back-compatibility
```

Comment:
the next file format increment, I guess. If somebody somewhere has a VDS with a sample as a variant or sample annotation, this line is required to maintain compatibility